alerts/openshift-virtualization-operator/CDIMultipleDefaultVirtStorageClasses.md

```diff
@@ -0,0 +1,33 @@
+# CDIMultipleDefaultVirtStorageClasses
+
+## Meaning
+
+This alert fires when more than one storage class has the annotation
+`storageclass.kubevirt.io/is-default-virt-class`.
```

Comment:
from this description, it implied that it's enough for the annotation `storageclass.kubevirt.io/is-default-virt-class` to **exist** in the StorageClass, but in the diagnosis step we're looking for storage classes in which that annotation equals to `"true"`.
so it is unclear whether the "true" value of the annotation makes any effect or not.

Reply:
Good point. WIll update.